src/providers/ipa/ipa_subdomains.c

```diff
@@ -692,7 +694,9 @@ ipa_subdom_get_k5_svc(struct ipa_subdomains_ctx *ctx,
                                         ctx->be_ctx,
                                         "IPA",
                                         dom->realm,
-                                        use_kdcinfo);
+                                        use_kdcinfo,
```

Comment:
I don't think the changes in ipa_subdomains.c are needed (and I'm sure they don't do anything :-))

The IPA subdomains provider is a bit special with respect to the kdcinfo files. The kdcinfo files here are currently not written as a reaction to resolving a fail over service, but just at startup as either ad_site in a subdomain section is resolved or just host names. And the writing of the kdcinfo files as the final step is done in `ipa_subdomains_write_kdcinfo_write_step`. So i would just pass -1 or whatever currently translates to "unlimited" and would add a comment that this is for now dummy code until/if ever the kerberos service for trusted domains on IPA client is represented by a real fail over service.